src/frontend/index.ts

```diff
@@ -15,7 +15,7 @@ function secureHeaders(router: Router, frontendApp: Express) {
 	router.use(helmet.contentSecurityPolicy({
 		useDefaults: true,
 		directives: {
-			defaultSrc: ["'none'"],
+			defaultSrc: ["'self'"],
```

Comment:
AUI image components don't render when this is set to "none". This is not prohibited by the [CSP guidelines](https://hello.atlassian.net/wiki/spaces/PMP/pages/139161777/Guideline+-+Content+Security+Policy) but would love suggesstions if anyone has a better alterative. Error in the console when this is set to "none" is `Refused to load ... because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`